test/test.js

```diff
@@ -285,6 +329,68 @@ describe('justified-layout', function () {
 
 		});
 
+		it('should return widows with a left layout through the default', function () {
+
+			var geometry1LeftWidow = justifiedLayout([1, 1, 1, 1]);
+
+			expect(geometry1LeftWidow.boxes[0].left).toEqual(geometry1LeftWidow.boxes[3].left);
+
+		});
+
+		it('should return widows with a specified left layout', function () {
+
+			var geometry1LeftWidow = justifiedLayout([1, 1, 1, 1], {
+				widowLayoutStyle: 'left'
+			});
+
+			expect(geometry1LeftWidow.boxes[0].left).toEqual(geometry1LeftWidow.boxes[3].left);
+
+		});
+
+		testLayoutsForCenteredWidows.forEach(function (layout) {
+
+			it('should return widows with a centered layout for [' + layout.toString() + ']', function () {
```

Comment:
Is this normal, printing out dynamic information with the test definition? I like it.